src/EPFL_Core_Command.php

```diff
@@ -301,22 +301,57 @@ public function symlink ($args = array(), $assoc_args = array()) {
 
             if(!file_exists($image_element))
             {
-                \WP_CLI::warning("Image element doesn't exists (".$image_element."), skipping site symlink procedure", true);
+                \WP_CLI::warning("Image element doesn't exist (".$image_element."), skipping site symlink procedure", true);
 
                 return;
             }
 
-            /* We rename file/folder if requested, or delete it */
-            $this->delete_or_copy_original_file_folder($site_element, true);
-
-            if(!symlink($image_element, $site_element))
+            if(!$this->ensure_symlink($image_element, $site_element, /* $remove_if_needed = */ TRUE))
             {
                 \WP_CLI::error("Cannot create symlink from '".$site_element."' to '".$image_element."'", true);
             }
         }
         /* Going back to original working directory  */
         chdir($current_wd);
     }
+
+    private function ensure_symlink ($from, $to, $remove_if_needed=FALSE) {
```

Comment:
Est-ce que tu pourrais utiliser `$target` et `$link` pour remplacer respectivement `$from` et `$to` ?
Parce que là, la lecture du code n'est pas logique... surtout quand on arrive à la fonction `symlink` ... 
Je continuerai la review une fois le changement fait car là je m'y perds... merci

Reply:
Fait dans f732190. (Tu sais que tu peux pousser dans la branche de code review si jamais ? ...)